solr/modules/jwt-auth/README.md

```diff
@@ -0,0 +1,25 @@
+Apache Solr JWT Authentication Plugin
+=====================================
+
+Introduction
+------------
+Solr can support [JSON Web Token](https://en.wikipedia.org/wiki/JSON_Web_Token) (JWT) based 
+Bearer authentication with the use of the JWTAuthPlugin.
+
+This allows Solr to assert that a user is already authenticated with an external 
+[Identity Provider](https://en.wikipedia.org/wiki/Identity_provider) by validating 
+that the JWT formatted [access token](https://en.wikipedia.org/wiki/Access_token) 
+is digitally signed by the Identity Provider.
+
+The typical use case is to integrate Solr with an [OpenID Connect](https://en.wikipedia.org/wiki/OpenID_Connect) 
+enabled IdP.
```

Comment:
nit: is `IdP` a well known term for Identity Provider? Maybe wouldn't hurt to spell it out here, especially for somebody new that is exploring our contrive as a measure of project or maturity.

Reply:
Spelling it out now.